contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -45,6 +48,13 @@ def register_options(cls, register):
                  NodeDistribution.VALID_PACKAGE_MANAGER_LIST.keys()))
       register('--yarnpkg-version', advanced=True, default='v0.19.1', fingerprint=True,
                help='Yarnpkg version. Used for binary utils')
+      register('--eslint-setupdir', advanced=True, fingerprint=True,
+               help='Find the package.json under this dir for installing eslint and plugins.')
+      register('--eslint-config', advanced=True, fingerprint=True,
```

Comment:
Should mark this option as`type=file_option` (which will cause it to be deeply fingerprinted).
```
from pants.option.custom_types import file_option
```